.cirrus.yml

```diff
@@ -34,3 +40,37 @@ multi_arch_build_task:
   build_script:
     - echo "Build the ${STAGING_IMAGE_NAME}:${CURRENT_TAG} image supporting two architectures, linux/amd64 and linux/arm64"
     - docker buildx build --platform linux/amd64,linux/arm64 --tag "${STAGING_IMAGE_NAME}:${CURRENT_TAG}" --push "${CURRENT_MAJOR_VERSION}"
+
+private_scan_task:
+  env:
+    platform: amd64
+  ec2_instance:
+    <<: *VM_TEMPLATE
+  login_script:
+    - docker login --username $DOCKER_USERNAME --password $DOCKER_PASSWORD
+  setup_script:
+    - apt-get update && apt-get install -y --no-install-recommends openjdk-17-jre
```

Comment:
It is better to do the same as I did at https://github.com/SonarSource/docker-sonarqube/pull/641 to avoid the flaky CI pipeline.

Reply:
makes sense, done.